src/main/java/ua/artcode/billapp/controller/CompanyController.java

```diff
@@ -32,15 +32,15 @@ public CompanyController(CompanyService companyService) {
     }
 
     //TODO: replace request param "id by "company" with type... Company.class (how unexpected, huh?)
-    @RequestMapping(path = "/get-opened-bills", method = RequestMethod.GET)
+    @RequestMapping(path = "/rest/get-opened-bills", method = RequestMethod.GET)
```

Comment:
What is the meaning of the subpath "rest"? You wanna integrate SOAP also?

Reply:
Oh, now I see. "csrf" filter. Niiiiiice.

Reply:
I didn't find another way to separate ordinary endpoints from /token.

Reply:
Ordinary? What do you mean when you say "ordinary endpoints"?

Reply:
I wrote all my comments and explanations in slack